### PR TITLE
logdy: init at 0.13.0

### LINKED ISSUE #329883: Package request: logdy (go)
**Project description**

terminal logs in web browser

**Metadata**

* homepage URL: [https://logdy.dev/](https://logdy.dev/)
* source URL: [https://github.com/logdyhq/logdy-core](https://github.com/logdyhq/logdy-core)
* license: [Apache-2.0 license](https://github.com/logdyhq/logdy-core#Apache-2.0-1-ov-file)
* platforms: darwin, linux, windows

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by Sigmanificient**
## Description of changes

Fixes #329883

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by ByteSudoer — APPROVED** [review]

File: pkgs/by-name/lo/logdy/package.nix [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  # After the build this derivation will generate two binaries.
+  # The first one is getting renamed besed on the documentation
```
Comment (donovanglover) [inline pkgs/by-name/lo/logdy/package.nix:29]:
```suggestion
  # The first one is getting renamed based on the documentation
```

File: pkgs/by-name/lo/logdy/package.nix [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  # The second one is just to launch a demo. This functionnality could be achieved with the first one
```
Comment (donovanglover) [inline pkgs/by-name/lo/logdy/package.nix:30]:
```suggestion
  # The second one is just to launch a demo. This functionality could be achieved with the first one
```

**Review by donovanglover — COMMENTED** [review]
Tested binary and works as expected

**Comment by SuperSandro2000** [comment]
@ofborg build logdy

**Pull request merged**